.gitignore

```diff
@@ -43,3 +43,8 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+# Docker ignores
+key.properties
+keystore.jks
+.env
```

Comment:
Just a note, we removed the need to have a `env` file because there's no use for it at the moment in https://github.com/thunder-app/thunder/pull/946. I think we can remove that here too!